src/main/java/pp/app/weather/WeatherService.java

```diff
@@ -0,0 +1,38 @@
+package pp.app.weather;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+@Service
+public class WeatherService {
+
+    private final String URL = "http://api.openweathermap.org/data/2.5/weather?q=";
+    private final String KEY = "&units=metric&appid=edb5454f20d34bda28ca69ec76f46b9c";
```

Comment:
Jak robisz stałe to brakuje tutaj jeszcze static. Ale ja bym bardziej pobawił się obiektem `URI` - będzie wg mnie czytelniej - oczywiśćie nie narzucam sposobu